tests/integration/defs/perf/test_perf_sanity.py

```
@@ -1490,6 +1510,12 @@ def run_cmd(self, server_idx: int) -> List[str]:
         elif self.disagg_serving_type == "DISAGG_SERVER":
             try:
                 self._generate_disagg_server_config(server_idx)
+                # The config carries port 0; publish the resolved address so
+                # the BENCHMARK task can find the server.
+                disagg_cmd = disagg_cmd + [
+                    "--report_addr",
+                    self._disagg_server_addr_file(server_idx),
+                ]
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)